### DISCUSSION TITLE
多选select

### DISCUSSION THREAD

**Discussion by lanrenbulan** [layui/layui#1597] [分享]
## 功能点
* 支持多选
* 支持搜索
* 支持折叠

## 在线预览
前往[仓库](https://github.com/lanrenbulan/layui-multiple-select)

## 其他

如果对你有需要，感谢star!!!

**Comment by LSL1618** [comment]
示例效果挺好的！

**Comment by LSL1618** [comment]
如果可以，添加支持以输入内容创建新选项。
